### ISSUE TITLE
Clear pk

### ISSUE THREAD

**Issue by cfytrok** [mathandpencil/django-bulk-sync#10]
What's the point of clearing pk before saving new object?
```
if old_obj is None:
    # This is a new object, so create it.
    # Make sure the primary key field is clear.
    new_obj.pk = None
    new_objs.append(new_obj)
```

I get list of object from other server using REST API. I want to save them with ids that server provide. And server id is primary key, because it is used in foreign key.

**Comment by ses4j** [comment]
Hmm, I agree I don't think it's the right behavior, and cannot come up with a good argument for it, though it was clearly intentional.

In your use case, are you getting both objects-needing-INSERT and objects-needing-UPDATE from this API?

**Comment by ses4j** [comment]
@cfytrok See the `keep_pk` branch and let me know if that makes sense to you.

**Comment by gannonprudhomme** [comment]
Would also like this added! (branch looks good to me 😄). Adding the changes in that branch makes this function how I would expect for my models' setup.

**Comment by tanabi** [comment]
Please incorporate this feature!  I'd love to use this library but I can't because of this.

**Comment by ses4j** [comment]
@tanabi @gannonprudhomme Released as v3.0.0 due to nature of breaking change, though idk that it'll affect people much.

**Comment by tanabi** [comment]
Thanks so much!
